src/gogo.py

```diff
@@ -73,7 +74,7 @@ def get(self):
             return '"name" param is required', 400
         shortcut = Shortcut.query.filter(Shortcut.name == name).first()
         if not shortcut:
-            return flask.redirect('/?error=%s+does+not+exist' % name)
+            return flask.redirect(f'{HTTPS_REDIRECT_URL}/?error={name}+does+not+exist')
```

Comment:
@eshaverma It is quite boilerplate code to append the app base url whenever the redirect happens, is there a way to make it concise?